packages/react-topology/src/components/defs/__tests__/__snapshots__/SVGDefs.spec.tsx.snap

```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`SVGDefs should get #addDef and #removeDef from context 1`] = `<DocumentFragment />`;
```

Comment:
this is an empty snapshot? is it service a purpose?

Reply:
Looked closer at the component and it always returns null. I'm going to end up deleting this test as well and adding it to the list to follow-up on. 

This wouldn't be a candidate for an RTL test since it's basically just verifying context works (provider, consumer, verify the consumer gets what the provider passes). Where yes, it's possible to setup context incorrectly, but type checking combined with already tested react functionality makes me think this is overkill and not necessary.

Reply:
Managed to come up with something for this, to bring back the Setter test, and the one from transformers. Please see updates in latest commit.

Reply:
Long story short for the SVGDefs test; Verifying functions are called on mount and on unmount implicitly prove the context props are passed down appropriately.